src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -180,14 +182,25 @@
      * Little class to hold the sequence of a pair of reads and tile location information.
      */
     static class PairedReadSequence extends PhysicalLocationShort {
+
+        static final  int NUMBER_BASES_IN_READ = 150;
```

Comment:
"EXPECTED_BASES_IN_READ"? please add a note to make clear if the code will function with a different (larger or smaller) read length and what will be the effect.